internal/factsengine/gatherers/registry.go

```diff
@@ -36,8 +77,32 @@ func (m *Registry) AddGatherers(gatherers map[string]FactGatherer) {
 	m.gatherers = result
 }
 
-func NewRegistry(gatherers map[string]FactGatherer) *Registry {
-	return &Registry{
-		gatherers: gatherers,
+func (m *Registry) getLatestVersionForGatherer(name string) (string, error) {
```

Comment:
Done, but I put the function on top of the registry struct declaration, because I don't want to interrupt visually the functions that are defined as method on the struct receiver